server/master/src/main/java/org/apache/accumulo/master/tableOps/Utils.java

```diff
@@ -59,21 +57,16 @@ static void checkTableDoesNotExist(Instance instance, String tableName, Table.ID
       throw new AcceptableThriftTableOperationException(null, tableName, operation, TableOperationExceptionType.EXISTS, null);
   }
 
-  static <T extends AbstractId> T getNextTableId(String tableName, Instance instance, Class<T> idClassType) throws AcceptableThriftTableOperationException {
+  static String getNextTableId(String tableName, Instance instance) throws AcceptableThriftTableOperationException {
```

Comment:
If you wanted to keep the original return type, you could pass in a function to map from String to ID:

```java
static <T extends AbstractId> T getNextId(String name, Instance instance, Function<String, T> newIdFunction) throws AcceptableThriftTableOperationException {
    // ...
    return newIdFunction.apply(new String(nid, UTF_8));
}
```

Can use it like this:
```java
Table.ID tableId = Util.getNextId(tableName, instance, Table.ID::of);
```